packages/vaults/src/transaction/member/member_update_role.ts

```diff
@@ -10,8 +10,20 @@ import {candidToRole, roleToCandid} from "../../util/helper";
 import {RequestMapperAbstract} from "../request_mapper";
 
 
+/**
+ * Interface for a transaction that updates role of an existing member.
+ * The memberId is the principal of the user with the default subaccount.
+ * This transaction can only be requested or approved only by users with the admin role.
+ * This transaction can be executed in a batch
```

Comment:
This transaction can only be requested or approved by users with the admin role.
This transaction can be executed in a batch, which means that all transactions marked with a single batch_id are executed or rejected together.